File: Strings/__3000__/3088/3088.py
def fiexText(text):
    fixed = ""
    for i in range(len(text)):
        char = text[i]
        if i < len(text) - 1:
            next = text[i + 1]
            if char == " " and (next == "," or next == "."):
                continue
        fixed += char
    return fixed

while True:
    try:
        line = input()
        print(fiexText(line))
    except EOFError:
        break

File: Strings/__3000__/3088/3088.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split("\n")

const fixText = (text) => {
    let fixed = ""
    for(let i = 0; i < text.length; i++){
        const char = text[i];
        const next = text[i + 1];
        if(char === " " && (next === "." || next === ",")){
            continue;
        }
        fixed += char;
    }
    return fixed;
}

console.log(input.map(fixText).join("\n"))